tests/e2e/test_api.py

```diff
@@ -687,33 +697,39 @@ def test_feedback() -> None:
         },
         timeout=BASIC_ENDPOINTS_TIMEOUT,
     )
-    assert posted_feedback.status_code == requests.codes.ok
-    assert posted_feedback.json() == {"response": "feedback received"}
-
-    # check the feedback store has one feedback
-    stored_feedback = client.get("/v1/feedback/list", timeout=BASIC_ENDPOINTS_TIMEOUT)
-    assert stored_feedback.status_code == requests.codes.ok
-    assert "feedbacks" in stored_feedback.json()
-    assert len(stored_feedback.json()["feedbacks"]) == 1
-
-    # remove the feedback
-    remove_feedback = client.delete(
-        f'/v1/feedback/{stored_feedback.json()["feedbacks"][0]}',
-        timeout=BASIC_ENDPOINTS_TIMEOUT,
-    )
-    assert remove_feedback.status_code == requests.codes.ok
-    assert remove_feedback.json() == {"response": "feedback removed"}
 
-    # check the feedback store is empty again
-    removed_feedback = client.get("/v1/feedback/list", timeout=BASIC_ENDPOINTS_TIMEOUT)
-    assert removed_feedback.status_code == requests.codes.ok
-    assert "feedbacks" in removed_feedback.json()
-    assert len(removed_feedback.json()["feedbacks"]) == 0
+    assert response.status_code == requests.codes.ok
+
+    assert feedback_dir.exists()
+
+    feedbacks = list(feedback_dir.glob("*.json"))
+    assert len(feedbacks) == 1
+
+    feedback = feedbacks[0]
+    with open(feedback) as f:
+        feedback_data = json.load(f)
+
+    assert feedback_data["user_id"]  # we don't care about actual value
+    assert feedback_data["conversation_id"] == CONVERSATION_ID
+    assert feedback_data["user_question"] == "what is OCP4?"
+    assert feedback_data["llm_response"] == "Openshift 4 is ..."
+    assert feedback_data["sentiment"] == 1
 
 
 @pytest.mark.cluster()
-def test_feedback_can_post_with_wrong_token():
-    """Test posting feedback with improper auth. token."""
+def test_feedback_storing__cluster():
+    """Test if the feedbacks are stored properly."""
+    feedbacks_path = "/app-root/ols-user-data/feedback"
```

Comment:
prob. refactor it as a module constant?

Reply:
If it would be used in more than one test, I would.